qutip/floquet.py

```diff
@@ -100,15 +99,11 @@ def floquet_modes(H, T, args=None, sort=False, U=None):
         U = propagator(H, T, [], args)
 
     # find the eigenstates for the propagator
-    evals, evecs = la.eig(U.full())
+    evals, evecs = U.eigenstates()
```

Comment:
I am not well aware of the internal code of QuTiP, but it seemed to me as I read the code that `Qobj.eigenstates` will do the same as the previous line. Using it makes the eigenmodes ordering simpler at the end of the function call. Plus, I guess it is better to use a single method to compute eigenstates across the whole library, so that any part of the library will take advantage of potential enhancements made in eigenstates computation in the future.